### PR TITLE
Fix StringBuilder param names for no more capacity on netfx

### PR DESCRIPTION
**Pull request by hughbe**
Contributes to #17723

### CODE REVIEW

File: src/System.Runtime/tests/System/Text/StringBuilderTests.cs [hunk 1]
```diff
@@ -454,8 +455,8 @@ public static void Append_UInt_NoSpareCapacity_ThrowsArgumentOutOfRangeException
         {
             var builder = new StringBuilder(0, 5);
             builder.Append("Hello");
-
-            Assert.Throws<ArgumentOutOfRangeException>("valueCount", () => builder.Append((uint)1)); // New length > builder.MaxCapacity
+
```
Comment (joperezr) [inline src/System.Runtime/tests/System/Text/StringBuilderTests.cs]: NIT: indentation

File: src/System.Runtime/tests/System/Text/StringBuilderTests.cs [hunk 2]
```diff
@@ -531,23 +537,34 @@ public static unsafe void Append_CharPointer(string original, char[] charArray,
...
+        {
+            var builder = new StringBuilder(0, 5);
+            builder.Append("Hello");
+
+            Assert.Throws<ArgumentOutOfRangeException>(s_noCapacityParamName, () =>
             {
-                fixed (char* value = new char[] { 'a' }) { builder.Append(value, 1); } // New length > builder.MaxCapacity 
-        });
+                fixed (char* value = new char[] { 'a' }) { builder.Append(value, 1); }
+            });
```
Comment (joperezr) [inline src/System.Runtime/tests/System/Text/StringBuilderTests.cs]: NIT: same here

Comment (hughbe) [inline src/System.Runtime/tests/System/Text/StringBuilderTests.cs]: I think this is an improvement on indentation. Previously:
```
            Assert.Throws<ArgumentOutOfRangeException>(s_noCapacityParamName, () =>
            {
                fixed (char* value = new char[] { 'a' }) { builder.Append(value, 1); }
        });
```

File: src/System.Runtime/tests/System/Text/StringBuilderTests.cs (hunk 1 above)
Comment (hughbe) [inline src/System.Runtime/tests/System/Text/StringBuilderTests.cs]: fixed, thanks

**Comment by danmosemsft** [comment]
@dotnet-bot test innerloop ubuntu16.04 arm debug cross build (`  Aborted (core dumped)
/opt/corefx/Tools/packageresolve.targets(35,5): error MSB3073: The command ""/opt/corefx/Tools/dotnetcli/dotnet" restore --no-cache --packages "/opt/corefx/packages" --source https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json --source https://dotnet.myget.org/F/dotnet-core/api/v3/index.json --source https://api.nuget.org/v3/index.json "/opt/corefx/bin/obj/Unix.AnyCPU.Debug/ilasm/netcoreapp/Unix-arm/project.json"" exited with code 134. [/opt/corefx/external/ilasm/ilasm.depproj]`)

**Comment by danmosemsft** [comment]
thanks @hughbe

**Pull request merged**